packages/core/src/store/feed-aggregation-store.ts

```diff
@@ -69,9 +79,38 @@ export class FeedAggregationStore extends ObjectStore<number, StreamID> {
     }
   }
 
+  streamIDs(gt?: string): ReadableStream<StreamID> {
+    const source = new StreamIDFeedSource(this.find, this.onWrite.asObservable(), gt)
+    return new ReadableStream(source)
+  }
+
   async close(): Promise<void> {
     clearInterval(this.#interval)
     await this.tasks.onIdle()
     await super.close()
   }
 }
+
+class StreamIDFeedSource implements UnderlyingSource<StreamID> {
+  private gt: string | undefined
```

Comment:
What does `gt` stands for?

Reply:
I thought it is a traditional name for "greater than".
- `gt` — greater than,
- `lt` — less than,
- `gte` or `ge` — greater than or equal to,
- `lte` or `le` — less than or equal to,
- `eq` — equal to.

Reply:
yes it is, but at first glance its not very clear what is getting saved here, Im assuming its a streamId since its used to `find` but I could be wrong, can you clarify what will be stored here?